CONTRIBUTING.md

```diff
@@ -1,35 +1,40 @@
-# Contributing to Sentry SDK for Laravel
+<p align="center">
+  <a href="https://sentry.io/?utm_source=github&utm_medium=logo" target="_blank">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-wordmark-dark-280x84.png" alt="Sentry" width="280" height="84">
+  </a>
+</p>
 
-We welcome contributions to sentry-laravel by the community. See the [Contributing to Docs](https://docs.sentry.io/contributing/) page if you want to fix or update the documentation on the website.
+# Contributing to the Sentry SDK for Laravel
 
-## How to report a problem
+We welcome contributions to `sentry-laravel` by the community.
 
 Please search the [issue tracker](https://github.com/getsentry/sentry-laravel/issues) before creating a new issue (a problem or an improvement request). Please also ask in our [Sentry Community on Discord](https://discord.com/invite/Ww9hbqr) before submitting a new issue. There is a ton of great people in our Discord community ready to help you!
 
-If you feel that you can fix or implement it yourself, please read a few paragraphs below to learn how to submit your changes.
+If you feel that you can fix or implement it yourself, please read on to learn how to submit your changes.
 
 ## Submitting changes
 
 - Setup the development environment.
-- Clone sentry-laravel and prepare necessary changes.
+- Clone the `sentry-laravel` repository and prepare necessary changes.
 - Add tests for your changes to `tests/`.
 - Run tests and make sure all of them pass.
-- Submit a pull request, referencing any issues it addresses.
+- Submit a pull request, targeting the `develop` branch if you added any new features. For bug fixes of the current release, please target the `master` branch instead.
```

Comment:
@stayallive This is entirely your call. Whatever works best 🙂

Reply:
There is no `develop` branch here, for now going to `master` has always worked here :)

Maybe this should change in the future... but for now all PR's need to go to `master`.